internal/pkg/policy/parsed_policy_test.go

```diff
@@ -11,107 +11,65 @@ import (
 	"encoding/json"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/elastic/fleet-server/v7/internal/pkg/model"
 )
 
 func TestNewParsedPolicy(t *testing.T) {
 	// Run two formatting of the same payload to validate that the sha2 remains the same
-	payloads := []string{
-		testPolicy,
-		minified,
-	}
-
-	for _, payload := range payloads {
-		// Load the model into the policy object
-		var m model.Policy
-		if err := json.Unmarshal([]byte(payload), &m); err != nil {
-			t.Fatal(err)
-		}
-
-		m.Data = json.RawMessage(testPolicy)
-
-		pp, err := NewParsedPolicy(context.TODO(), nil, m)
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		fields := []string{
-			"id",
-			"revision",
-			"outputs",
-			"output_permissions",
-			"agent",
-			"inputs",
-			"fleet",
-		}
-
-		// Validate the fields;  Expect the following top level items
-		if len(pp.Fields) != len(fields) {
-			t.Error("Expected N fields")
-		}
-
-		for _, f := range fields {
-			if _, ok := pp.Fields[f]; !ok {
-				t.Errorf("Missing field %s", f)
-			}
-		}
-
-		// Now validate output perms hash
-		if len(pp.Roles) != 1 {
-			t.Error("Only expected one role")
-		}
-
-		// Validate that default was found
-		if pp.Default.Name != "other" {
```

Comment:
I'm not sure how this test was passing as the minified policy had a single output named "default"